### PR TITLE
fix `JSZip Returns an empty blob when ZipObject.async used` (close #2115)

### LINKED ISSUE #2115: JSZip Returns an empty blob when ZipObject.async used
### What is your Scenario?

 Our app uses jszip to uncompress files from a zip. We get this file through a request and later try to unzip it using jszip.
At a certain point this command is run :

**def.async('blob').then(content => new ContentData(path, content)**

where def is a ZipObject. 

### What is the Current behavior?

 When using our app without testcafe everything works properly, it returns a blob with correct filesize. 
When using our app through testcafe the blob returned is size 20 and it turns out to be an empty string. It returns 20 whether the file is empty or it contains content. Each file provided from the zip file returns 20 and finally an empty file.

This makes decompressing these files impossible through testcafe's hammerhead.

### What is the Expected behavior?

 Decompressing with JSZip on testcafe should first get the correct size when using async on a jszip ZipObject of type binary
### What is your public web site URL?

Your website URL (or attach your complete example): 

<details>
<summary>Your complete app code (or attach your test files):</summary>

 
```js
 **def.async('blob').then(content => new ContentData(path, content)**

```
</details>

<details>
<summary>Screenshots:</summary>

```

```
</details>

### Steps to Reproduce:

1. Get a zip file over a request
2. Use jszip to filter through the files inside
3. Finally use async function on zipobjects to get the size and type of each.

### Your Environment details:

* node.js version:  10.15.1                    
* browser name and version: Chrome 76 
* platform and version: Mac Os          
* other:

### PR DESCRIPTION
**Pull request by Farfurix**
https://github.com/DevExpress/testcafe-hammerhead/issues/2115

### Changes
1. Process only the "`Array<string | number | boolean>`" `Blob` parts array case.

### CODE REVIEW

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

File: src/client/sandbox/node/window.ts [hunk 1]
```diff
@@ -124,6 +124,23 @@ export default class WindowSandbox extends SandboxBase {
         return stack;
     }
 
+    private static _isProcessable (blobParts: Array<any>) : boolean {
```
Comment (LavrovArtem) [inline src/client/sandbox/node/window.ts]:
```suggestion
    private static _isProcessableBlob (parts: Array<any>) : boolean {
```

File: test/client/fixtures/sandbox/node/classes-test.js [hunk 2]
```diff
@@ -120,6 +122,78 @@ test('window.Blob([data], { type: "" }) should return correct result for `ArrayB
     ]);
 });
 
+asyncTest('should process Blob parts in the case of the "Array<string | number | boolean>" array (GH-2115)', function () {
+    var scriptForProcessing = ['self.onmessage = function() { var t = {};', '__set$(t, "blobTest", ', 1, '+', true, '); postMessage(t.blobTest); };'];
```
Comment (LavrovArtem) [inline test/client/fixtures/sandbox/node/classes-test.js]:
```suggestion
    var parts = [
        'self.onmessage = function() { var t = {blobTest:',
        1,
        '+',
        true,
        '+ location.toString()}}'];
```

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Comment by Farfurix** [comment]
@testcafe-build-bot retest

**Pull request merged**